lib/minimap-element.coffee

```diff
@@ -267,6 +267,7 @@ class MinimapElement extends HTMLElement
 
     @applyStyles @controls,
       width: Math.min(@canvas.width, @width) + 'px'
+      'margin-left': (@marginRight || 0) + 'px'
```

Comment:
More than a fix, I really need a reproducible test case for the test suite. As I already explained, the solution IMO is not to add margin to the controls div. That div should always have the size of the smallest element in the minimap (either the `atom-text-editor-minimap` or `canvas` element), this is the line 269 above. If this doesn't work in some situation, the solution should be to fix this, not to add another offset somewhere.

In your screen above, does the controls div have a style attribute?

Also, do you see in the following test anything that doesn't match the configuration in which it should fail?

https://github.com/fundon/atom-minimap/blob/feature/quick_settings_custom_element/spec/minimap-element-spec.coffee#L702-L731